src/main/java/org/apache/accumulo/examples/client/SequentialBatchWriter.java

```diff
@@ -53,28 +53,16 @@ public static Value createValue(long rowId) {
     private static final BatchWriterConfig BWDEFAULTS = new BatchWriterConfig();
 
     @Parameter(names = {"-t"}, required = true, description = "table to use")
-    public String tableName;
+    public String tableName = "batch";
```

Comment:
Do we want to hardcode the table name or allow the user to define it using -t as the example demonstrates?

Reply:
The hardcoded table name in this case is the default.  If the user does not set it on the command line with the `-t` option, it will be `batch`.